simulator/metrics.go

```diff
@@ -0,0 +1,31 @@
+package simulator
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	prometheusmetrics "github.com/deathowl/go-metrics-prometheus"
```

Comment:
Heads up, the client libs appear to be going stale. 

[go-metrics-prometheus](https://github.com/deathowl/go-metrics-prometheus/pull/15) not merging things because PR's rotting at github.com/rcrowley/go-metrics.

Reply:
Ouch, good call.  Will likely be switching prometheus libs soon and won't likely stick with go-metrics, so will address then.